### ISSUE TITLE
Intent label size PhoATIS

### ISSUE THREAD

**Issue by hieudx149** [VinAIResearch/JointIDSF#5]
Hi, 
I want to ask about intent label size in PhoATIS dataset, in paper you have shown that PhoATIS consists of 28 intent labels. Is this the size of the intent label in the training set or on whole dataset (including dev set and test set) ?
